packages/yarnpkg-cli/sources/polyfills.ts

```diff
@@ -1,6 +1,10 @@
+// Remove when dropping Node 10 (~April 2021)
+import 'array-flat-polyfill';
```

Comment:
We already have `lodash/flatten` in the bundle so you can use that instead and save 647 bytes ^^